jboss-kie-smartrouter/tests/bats/jboss-kie-smartrouter.bats

```diff
@@ -51,3 +57,58 @@ teardown() {
     unset JAVA_INITIAL_MEM_RATIO
     unset JAVA_MAX_MEM_RATIO
 }
+
+@test "verify the env vars when LOG_LEVEL is set" {
+    LOG_LEVEL="WARN"
+    LOGGER_CATEGORIES=org.xyz=INFO,com.acme=DEBUG
+    configure_logger_config_file
+    [[ "${LOG_LEVEL}" == WARN ]]
+    [[ "${JAVA_OPTS_APPEND}" ==  " -Djava.util.logging.config.file=/tmp/logging.properties" ]]
+    unset JAVA_OPTS
+}
+
+@test "verify the logger.properties with default values" {
+    configure_logger_config_file
+    file1=/tmp/logging.properties
```

Comment:
for a better reading replace all /tmp with $CONFIG_DIR.

Reply:
Ok